lsp-mode.el

```diff
@@ -7423,6 +7118,9 @@ returns the command to execute."
   (when (functionp 'lsp-ui-mode)
     (lsp-ui-mode))
 
+  (when lsp-headerline-breadcrumb-enable
+    (add-hook 'lsp-managed-mode-hook 'lsp-headerline-breadcrumb-mode))
```

Comment:
After rethinking, maybe this is better
```suggestion
    (lsp-headerline-breadcrumb-mode 1))
```
Since this is auto configure function and you don't want a hook left behind when cleaning `lsp-mode`

Reply:
This gives byte-compile errors because `lsp-mode.el` doesn't know `lsp-headerline.el`

Reply:
Maybe we should run another hook when cleaning `lsp-mode`

Reply:
Will it? Shouldn't `lsp-headerline-breadcrumb-mode` is autoloaded?
Or you can check for `functionp`, but I don't think it's necessary.

Reply:
Resolved this one, let's discuss in another comment thread